mllib-tests/v1p4/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -391,14 +392,21 @@ class NaiveBayesTest(sc: SparkContext)
 
     val threshold: Double = doubleOptionValue(THRESHOLD)
     val sf: Double = doubleOptionValue(SCALE)
+    val modelType = stringOptionValue(MODEL_TYPE)
```

Comment:
model type is always present : per your suggestion it was added to even the 1.3 in config.py.template